### ISSUE TITLE
Model.Band is missing

### ISSUE THREAD

**Issue by gamassimino** [nmaguirre/riocuartobandasderock#1]
Band.class contains the information about a band
attributes to consider...

```
public class Band {
    private Integer id;
    private List<Album> albumList;
    private List<BandMember> memberList;
    private List<Song> songList;
    private String name;
    private String release;
    private String end;
    private Genere genere;
}
```

where release is the creation date of the band and blablabla is the date end of band

we need to simulate a class Genere 

```
public class Genere {
	private Integer id;
	private String name;
	private String description;
	private String release;
	private List<String> initiatedBy;
	private List<Genere> influence;
```

**Comment by gamassimino** [comment]
fadcaa8
122b3c0

**Comment by ezedepetris** [comment]
first step of implementation [09fe396]

**Comment by gamassimino** [comment]
`Test's`
70c5f11
203140d
18ab0a5

**Comment by ezedepetris** [comment]
test's
[ 27653a4]

constructor's and delete extra attributes
[58332ba]

**Comment by lbuttignol** [comment]
@ezedepetris  @gamassimino The attribute `memberList` should be a `List<Artist>` because last Thursday decided to change `BandMember` to `Artist`.

On the other hand it was said that the `id` should be a `Strings` and are generated as follows
```
public Band( String a_name, Genere a_genere, List<BandMember> list_of_members, String a_release){
    id = UUID.randomUUID().toString(); //using UUID lib and hidden for the client
	name = a_name;
	genere = a_genere;
	memberList = list_of_members;
	release = a_release;
  }
```

**Comment by gamassimino** [comment]
Genere comments
0b36543

**Comment by ezedepetris** [comment]
BandModel and BandTest comments
[576d2a1]

**Comment by DanielSanchez94** [comment]
Fix your test because we changed the class Genre.

**Comment by ezedepetris** [comment]
Test of class Genre fixed [f40d760]
